lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -4,14 +4,38 @@ import { getResolver } from './test-resolver';
 
 export default TestModule.extend({
   init: function(componentName, description, callbacks) {
+    // Allow `description` to be omitted
+    if (!callbacks && typeof description === 'object') {
+      callbacks = description || {};
+      description = null;
+    }
+
     this.componentName = componentName;
+    this.isUnitTest = !!(callbacks.needs || callbacks.unit) ||
+      callbacks.integration === false;
```

Comment:
We discussed this in chat, but I'd like to default to being a unit test in the case that neither `needs` nor `integration` is present.  We can swap the default when we bump "major" versions.